src/components/teamBoxes/TeamOneBox.tsx

```diff
@@ -54,16 +54,11 @@ const TeamOneBox = () => {
             onDisconnect(playerOnTeamOneOperativesRef).remove();
             set(child(teamOneOperativesRef, playerId), { playerId, username });
             update(nestedPlayerRef, { teamId: team1Id });
-            dispatch(setTeamIdOnPlayer(team1Id))
```

Comment:
no longer need these in the team boxes since the onChildChanged listener is updating to redux for us